amqp/README.md

```diff
@@ -1,51 +1,41 @@
-AMQP 0.9.1 API Reference
-========================
+# API Reference
 
-If you don't know what AMQP is you can read [this](https://blogs.vmware.com/vfabric/2013/02/choosing-your-messaging-protocol-amqp-mqtt-or-stomp.html)
-and [this](https://www.rabbitmq.com/tutorials/amqp-concepts.html).
+TTN makes use of an AMQP Topic exchange (`amq.topic`). See [this](https://www.rabbitmq.com/tutorials/amqp-concepts.html) for details.
 
-TTN use the Topic AMQP exchange. The default exchange name is `amq.topic`
+AMQP is not yet available in the public network. The main reasons for this are: 
+_security_, _scalability_ and the AMQP protocol not being _standardised_ yet.
 
-__Public Network__
 
-*Building*
-
-## Routing keys behavior
-
-Routing keys act like filters. When a message arrive on the AMQP exchange
-the key is read and the message forwarded to all matching queue with the
-a matching routing queue.
-
-Routing keys make use of the wildcard `*` to match multiple queues.
-
-Example:
-* Match a device uplink: `<App_ID>.devices.<Dev_ID>.up`
-* Match all App devices uplink: `<App_ID>.devices.*.up`
-
-## Uplink
+* Host: The address of the handler on which your application is registered.
+* Port: `5672`
+* Exchange: `ttn.handler`
 
+## Uplink Messages
 
 **Routing key:** `<App_ID>.devices.<Dev_ID>.up`
```

Comment:
No underscores; this is the reference https://www.thethingsnetwork.org/docs/applications/mqtt/api.html